src/keys.ts

```diff
@@ -1,21 +1,21 @@
 export const isDarwin = /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
 
-export const KEYS =
-  {
-    ARROW_LEFT: "ArrowLeft",
-    ARROW_RIGHT: "ArrowRight",
-    ARROW_DOWN: "ArrowDown",
-    ARROW_UP: "ArrowUp",
-    ENTER: "Enter",
-    ESCAPE: "Escape",
-    DELETE: "Delete",
-    BACKSPACE: "Backspace",
-    CTRL_OR_CMD: isDarwin ? "metaKey" : "ctrlKey",
-    TAB: "Tab",
-    SPACE: " ",
-    QUESTION_MARK: "?",
-    F_KEY_CODE: 70,
-  } as const;
+export const KEYS = {
+  ARROW_LEFT: "ArrowLeft",
+  ARROW_RIGHT: "ArrowRight",
+  ARROW_DOWN: "ArrowDown",
+  ARROW_UP: "ArrowUp",
+  ENTER: "Enter",
+  ESCAPE: "Escape",
+  DELETE: "Delete",
+  BACKSPACE: "Backspace",
+  CTRL_OR_CMD: isDarwin ? "metaKey" : "ctrlKey",
+  TAB: "Tab",
+  SPACE: " ",
+  ES: "s",
```

Comment:
Why did you name it "ES" instead of S?

Reply:
Or just do `event.key = "s"` instead of adding a constant.